storage/innobase/include/srv0srv.h

```diff
@@ -985,6 +985,39 @@ bool
 srv_is_undo_tablespace(
 	ulint	space_id);
 
+/** Page gragmentation statistics */
+struct fragmentation_stats_t {
+	fragmentation_stats_t():
+		scan_pages_contiguous(0),
+		scan_pages_disjointed(0),
+		scan_pages_total_seek_distance(0),
+		scan_data_in_pages(0),
+		scan_garbage_in_pages(0)
+	{}
+
+	ulint scan_pages_contiguous;		/*!< number of contiguous
```

Comment:
"ulonglong" in server, "ulint" in InnoDB - maybe make them "ullint" in InnoDB to kind-of match?

Reply:
Good point. Hmm, not sure how I missed that.

Reply:
Refactored:  `fragmentation_stats_t` is now a struct in `sql_class.h`
introduced 2 new functions:
`thd_get_fragmentation_stats()` and `thd_add_fragmentation_stats()`
both InnoDB `export_var_t` and SQL `system_status_var` share single data structure `fragmentation_stats_t` definition